lib/dropdown_search.dart

```diff
@@ -2,6 +2,8 @@ library dropdown_search;
 
 import 'dart:async';
 
+// import 'package:platz_bank/dds/src/properties/clear_button_props.dart';
+// import 'package:platz_bank/dds/src/properties/dropdown_button_props.dart';
```

Comment:
Shouldn’t these unused imports be removed?

Reply:
Removed. But when expands attribute is true, then searchable widget fallbacks to easy text drawing because of I expected a bug, but I try fix this next day